app/views/helpers/lists.php

```diff
@@ -144,15 +160,24 @@ public function displayRow(
             );
             echo format(__('created by {listAuthor}', true),
                         array('listAuthor' => $link));
+            
+            echo $this->Html->tag(
+                'span',
+                ' '.$this->Date->ago($createdDate),
+                array(
+                    'class' => 'date'
+                )
+            );
```

Comment:
I got that; my point was that the date is currently aligned separately from the author. That means that all dates appear to be aligned in one col, no matter how long the author's name on its left side is. 
The names of the authors will differ and if the dates are not aligned separately, they will have different indentations - depending on the the length of the author's name. 
(I currently use td for alignment).